src/interchange/src/avro/encode.rs

```diff
@@ -119,20 +131,19 @@ pub struct AvroSchemaGenerator {
 
 impl AvroSchemaGenerator {
     pub fn new(
-        key_fullname: Option<&str>,
-        value_fullname: Option<&str>,
         key_desc: Option<RelationDesc>,
         value_desc: RelationDesc,
-        debezium: bool,
+        options: AvroSchemaOptions,
```

Comment:
I have a slight preference for destructuring here:

```suggestion
        AvroSchemaOptions {
            is_debezium,
            avro_value_nullname,
            avro_key_fullname,
            set_null_defaults
            }: AvroSchemaOptions,
```

That way, if anyone adds new items to that struct, they will be forced to think about whether they need to affect this function.

But it's a minor thing, so it's up to you whether to do it...

Reply:
I see what you mean, will update 👍